src/index.css

```diff
@@ -131,3 +161,57 @@ nav a {
   padding: 0.5em;
   font-size: 1em;
 }
+
+.categories h2 {
+  margin: 1em 0 0 2.5em;
+}
+
+.percentage {
+  width: 5.2em;
+  margin-left: 7em;
+  padding-left: 1em;
+}
+
+.percent {
+  font-size: 1.8em;
+  font-weight: 400;
+}
+
+.progress {
+  display: flex;
```

Comment:
- [ ] Hello @krisbobo, I notice repetition of `display: flex;` in all your container, One golden tip in styling is that we can create a class with any styling that is repeating and then give that class to any element we want. for example I can write
```CSS
.flex {
display: flex;
}
```
and then give `flex` class to any container I want. you can apply this rule to save a lot of lines in your CSS.